### ISSUE TITLE
You should not Administrator rights on a Windows machine to install Reflow

### ISSUE THREAD

**Issue by jtlafferty** [edge-reflow/issues#4]
Feedback from a customer:

I just got access today and tried installing the windows version of preview. However, I was unable to do so because of insufficient rights. Is there a reason the installer requires administrator rights, and if not could this be removed?

**Comment by mstade** [comment]
+1

It was me who submitted this originally. Happy to help with feedback if needed.

**Comment by jtlafferty** [comment]
I believe the problem here is that we had not signed the installers. We should be getting them signed for the next drop and hopefully this won't be a problem. Thanks again!

**Comment by mstade** [comment]
Cool, thanks for the update!

**Comment by jtlafferty** [comment]
Actually, I tested out some other Adobe apps including Illustrator and Edge Animate and you need Administrator rights to install them both. So, I'm moving this to a low priority, we may not fix this.

**Comment by mstade** [comment]
Is there any compelling reason why administrator rights are needed?

**Comment by jtlafferty** [comment]
I am not sure, but, I will ask.

**Comment by JeffryBooher** [comment]
Windows Requires that you be an Administrator to install anything.  This is a security feature of Windows. Only Administrators have access to things like the Program Files directory so this isn't really a bug.

**Comment by mstade** [comment]
@JeffryBooher that's not true. For instance, chrome installs without admin rights. It achieves this by installing into the user's directory. If a program doesn't _need_ admin rights &ndash; e.g. it doesn't have to be installed for every user &ndash; then it shouldn't ask for them.
